### DISCUSSION TITLE
Is there lima support the XDG spec?

### DISCUSSION THREAD

**Discussion by suspenss** [lima-vm/lima#2745] [Q&A] [answered]
Can i move the .lima dir to $XDG_CONFIG_HOME  or $XDG_DATA_HOME  from $HOME/.lima ? or can i move to the .config/ dir instead of in home dir?

**Comment by afbjorklund** [comment, marked as answer]
You can change the value of $LIMA_HOME, but the config and the data is mixed in the same instance directory so it does not follow the XDG spec

Note: it does use $XDG_CACHE_HOME for the download cache, though (see https://lima-vm.io/docs/dev/internals/ for details)

**Reply by suspenss** [reply]
Thanks!  I only read the configuration doc instead of dev doc, hah,  maybe i need search for dev doc firstly when i meet some new problem, hah.
very helpfull,   just keep home directory clean , ^_^

**Reply by afbjorklund** [reply]
It should probably mentioned in the user docs, especially if you have your home directory on NFS...
